config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-v2-config.yaml

```diff
@@ -509,8 +509,8 @@ presubmits:
               value: "true"
             - name: TEST_WINDOWS
               value: "true"
-            - name: WINDOWS_FLAVOR
```

Comment:
just remove following config since it's running on 2019 actually
```
            - name: WINDOWS_FLAVOR
              value: "containerd-2022"
```

Reply:
Done